src/AstCreate.fs

```diff
@@ -28,8 +28,8 @@ type LongIdentWithDots with
         sb.ToString()
 
 type SynPatLongIdentRcd with
-    static member Create (id, args) =
-        { Id = id; Args = args; Access = None; Range = range.Zero }
+    static member Create (id, args, ?extraId, ?typarDecls, ?access) =
```

Comment:
Since these fields are `option` types, I added them all as [optional parameters](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/parameters-and-arguments#optional-parameters).